src/helpers/script_helpers.sh

```diff
@@ -109,9 +109,15 @@ function run {
     undo_command
   else
     remove_step_files
+
+    # Running inside a subfolder can cause issues if that folder doesn't exist
+    # on the main branch. Hence we cd into the root directory of the Git repo here.
+    # See https://github.com/Originate/git-town/issues/371 for more details.
+    cd_git_root
     preconditions "$@"
     steps > "$STEPS_FILE"
     run_steps "$STEPS_FILE" undoable
+    cd_start_dir
```

Comment:
What about when we hit an error and then either need to abort and continue. I think we need to add the cd's to the steps list instead.